### ISSUE TITLE
add a way to see all the annotations on an image

### ISSUE THREAD

**Issue by dobkeratops** [bbernhard/imagemonkey-core#188]
it would be useful to be able to see the existing annotations on an image, e.g. when choosing what to annotate next, and just to see the progress accumulating;

perhaps this could be done in the 'add labels' view (which in conjunction with #187 could turn it into something of a 'hub' for the image) - e.g. a toggle (perhaps color code the labels, although past 7 it will certainly get hard to distinguish, but dont worry about that initially). it would probably be better to show these as a tinted filled overlay rather than outlines, because with many intertwined outlines it will be unclear which bounds an area .  Later you could worry about toggling individual annotations  for clarity in scenes with many labels, and many overlapping labels

it might also help illuminate the current gaps, further informing what labels would be useful to add

a view like this might help confirm when it's a safe assumption to say "the remaining labels fill the un-annotated area" - (e.g. consider an obvious case such as the images of just a dog on a lawn: after you annotate the dog, it's safe to assume ``the rest = grass``` .. there will be cases where, say, you could annotate the sky,road,pavement, and ```the rest=building```. you could choose strategically regarding how the outlines fit together and narrow down the remaining space
)

**Comment by bbernhard** [comment]
really cool idea, thanks a lot for sharing!

I'll start with that as soon as the annotation refinement (#178) is done. I think that shouldn't be too much work :)

**Comment by dobkeratops** [comment]
makes sense..  this can certainly wait.  getting annotation refinement & materials right is worth waiting for
